warp-tls/Network/Wai/Handler/WarpTLS.hs

```diff
@@ -129,6 +131,15 @@ data TLSSettings = TLSSettings {
     -- Default: Nothing
     --
     -- Since 3.2.2
+  , tlsSessionManagerConfig :: Maybe SM.Config
+    -- ^ Configuration for in-memory TLS session manager.
+    -- If Nothing, 'TLS.noSessionManager' is used.
```

Comment:
Why wouldn't we default to using some basic configuration? Is there a downside to turning on a manager by default?